src/dodal/devices/i10/rasor/rasor_scaler_cards.py

```diff
@@ -0,0 +1,12 @@
+from ophyd_async.core import Device
+
+from dodal.devices.current_amplifiers import StruckScaler
+
+
+class rasor_scaler_card_1(Device):
```

Comment:
Should: Use PascalCase for class names, see https://peps.python.org/pep-0008/#class-names

```suggestion
class RasorScalerCard1(Device):
```